### ISSUE TITLE
MaintenanceSolution.sql doesn't currently create all objects, as is claimed on the homepage

### ISSUE THREAD

**Issue by dansqldba** [olahallengren/sql-server-maintenance-solution#139]
The homepage suggests using MaintenanceSolution.sql to deploy the solution, as “This script creates all the objects and jobs that you need” but I haven’t found this to be the case today.

It does not seem to include the tables supporting new functionality for queues, so I had to download and deploy those scripts in addition to MaintenanceSolution.sql. 

The code hosted on GitHub also appears to be missing the CREATE TABLE statements for the new tables.

**Comment by olahallengren** [comment]
You are right. I haven't included the new Queue - objects in MaintenanceSolution.sql. It is on the to do list.

I was planning to wait with that until after I have moved the maintenance objects to its own schema, #91.
